### DISCUSSION TITLE
Why is Guild.owner typehinted as Optional[member]?

### DISCUSSION THREAD

**Discussion by ImNimboss** [Rapptz/discord.py#7338] [Q&A] [answered]
[Guild.owner in documentation](https://discordpy.readthedocs.io/en/master/api.html?highlight=Optional#discord.Guild.owner) is typehinted as Optional[member], but is the Optional[] necessary?
I'm not very sure if it's possible to have a guild without an owner, therefore I think the Optional[] is unnecesary. I may be wrong, feel free to correct me.

**Comment by imvaskel** [comment, marked as answer]
Intents can cause the owner to be None. (Specifically lack of intents.members)

**Reply by ImNimboss** [reply]
Oh ok thank you!

**Reply by Rapptz** [reply]
Also guild can have no owner, e.g. TnS deletes them.
